### PR TITLE
prevent warning when pcntl_fork() is on disable_functions php.ini bla…

### PR DESCRIPTION
**Pull request by staabm**
…cklist

| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | N/A 

`pcntl_fork` is blacklisted per default on ubuntu lts boxes. make sure deployer doesnt emit a warning in this case.

Without this fix you get
```
PHP Warning:  pcntl_fork() has been disabled for security reasons in phar:///usr/local/bin/dep/src/Utility/Reporter.php on line 22
```

### CODE REVIEW

File: src/Utility/Reporter.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ class Reporter
     public static function report(array $stats)
     {
         $pid = null;
-        if (extension_loaded('pcntl')) {
```
Comment (antonmedv) [inline src/Utility/Reporter.php]: Maybe it's better to use `function_exists`? If it available?

Comment (staabm) [reply src/Utility/Reporter.php]: `function_exists` also works. I feel it is less expressive, but nevertheless changed it.

**Comment by staabm** [comment]
Friendly ping @antonmedv

**Comment by staabm** [comment]
Fixed build

**Pull request merged**